src/tracer.c

```diff
@@ -227,8 +227,18 @@ handle_syscall(PROCESS_INFO *pi, const struct ptrace_syscall_info *entry,
 	    // int flags);
 	    record_process_start(pi->pid, pi->outname);
 	    break;
-	default:
```

Comment:
Why is this removed?

Reply:
@zvr Because it's redundant, after the switch the function exits anyway. Do we care about keeping it?